src/main/resources/application.properties

```diff
@@ -1,7 +1,7 @@
-spring.datasource.url=jdbc:mysql://localhost:3306/book_web?serverTimeZone=UTC
-spring.datasource.username=root
-spring.datasource.password=123456
-spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
+#spring.datasource.url=jdbc:mysql://localhost:3306/book_web?serverTimeZone=UTC
+#spring.datasource.username=root
+#spring.datasource.password=123456
+#spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
```

Comment:
Please do not leave commented code.
You may leave as it is for local run (in docker these properties will be overridden by compose anyway)
or remove them if not required